src/styles/getPopoverWithMeasuredContentStyles.ts

```diff
@@ -1,16 +1,9 @@
 import roundToNearestMultipleOfFour from './roundToNearestMultipleOfFour';
 import variables from './variables';
 
-/**
- * Compute the amount that the Context menu's Anchor needs to be horizontally shifted
- * in order to keep it from displaying in the gutters.
- *
- * @param {Number} anchorLeftEdge - Menu's anchor Left edge.
- * @param {Number} menuWidth - The width of the menu itself.
- * @param {Number} windowWidth - The width of the Window.
- * @returns {Number}
- */
-function computeHorizontalShift(anchorLeftEdge, menuWidth, windowWidth) {
+// Compute the amount that the Context menu's Anchor needs to be horizontally shifted
+// in order to keep it from displaying in the gutters.
```

Comment:
We should leave JSDoc, just remove types from it.
```suggestion
/**
 * Compute the amount that the Context menu's Anchor needs to be horizontally shifted
 * in order to keep it from displaying in the gutters.
 *
 * @param anchorLeftEdge - Menu's anchor Left edge.
 * @param menuWidth - The width of the menu itself.
 * @param windowWidth - The width of the Window.
 */
```

Reply:
@hayata-suenaga can you confirm?

Reply:
yep we just want to remove the type part from JSDoc comments. Yours look good

Reply:
updated